custom/icds_reports/reports/awc_reports.py

```diff
@@ -851,33 +875,38 @@ def get_data_for(query_class, filters):
                     'all': get_value(data, 'css_lactating_all'),
                     'format': 'percent_and_div',
                     'frequency': frequency
-                },
-                {
-                    'label': _(ADOLESCENT_GIRLS_ENROLLED_FOR_ANGANWADI_SERVICES),
-                    'help_text': _((
-                        "Of the total number of adolescent girls (aged 11-14 years), the percentage "
-                        "of girls enrolled for Anganwadi Services"
-                    )),
-                    'percent': percent_diff(
-                        'person_adolescent',
-                        data,
-                        prev_data,
-                        'person_adolescent_all'
-                    ),
-                    'color': get_color_with_green_positive(percent_diff(
-                        'person_adolescent',
-                        data,
-                        prev_data,
-                        'person_adolescent_all'
-                    )),
-                    'value': get_value(data, 'person_adolescent'),
-                    'all': get_value(data, 'person_adolescent_all'),
-                    'format': 'percent_and_div',
-                    'frequency': frequency
                 }
             ]
         ]
     }
+    if (not beta) or selected_month.date() >= ADOLESCENT_GIRLS_DATA_THRESHOLD:
```

Comment:
This changes awc report page tile
![awc_report_2](https://user-images.githubusercontent.com/21310920/71279471-db47f900-237e-11ea-961a-d01aebdac6e4.png)